config.coffee

```diff
@@ -118,7 +118,7 @@ module.exports =
   MARKETING_SIGNUP_MODAL_SLUG: 'miami'
   EOY_2016_ARTICLE: null
   EOY_2016_TEASER: 'https://artsy-vanity-files-production.s3.amazonaws.com/documents/year-in-art-teaser.html'
-  SHOW_ARTIST_CTA_CODE: null
+  SHOW_ARTIST_CTA_CODE: 'show-me'
```

Comment:
#minor but I shouldn't have merged with this change :)

doesn't matter tho /shrug